### PR TITLE
Update people.yml and chunmyong.park.md

### PR DESCRIPTION
**Pull request by cmpark0126**
Update meta information and profile page of Chunmyong Park.

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -24,6 +24,19 @@
     "(2019) B.S. in Computer Science and Engineering and Philosophy. Inha University.",
   ]
 
+- id: chunmyong.park
+  name: Chunmyong Park
+  korean_name: 박천명
+  status: present
+  title: Master Student
+  place: TBD
```
Comment (jeehoonkang) [inline _data/people.yml]: place: Rm. 4441, Bldg. E3-1, KAIST (+82-42-350-7878)

File: _data/people.yml [hunk 2]
```diff
@@ -24,6 +24,19 @@
...
+  github: cmpark0126
+  dblp: https://dblp.uni-trier.de/search?q=Chunmyong%20Park
```
Comment (jeehoonkang) [inline _data/people.yml]: You don't have a DBLP entry.  Please leave it empty.

**Comment by cmpark0126** [comment]
I modified people.yml as you said! Thank you.

**Comment by cmpark0126** [comment]
I modified people.yml as you said! Thank you.

**Comment by jeehoonkang** [comment]
- 감사합니다~
- @kaist.ac.kr 이메일 만들고 제게 알려주세요~

**Comment by cmpark0126** [comment]
네 교수님! 이메일 만들어 두었습니다!

chunmyong.park@kaist.ac.kr 입니다!

감사합니다!

**Pull request merged**